docs/docs/reference/new-types/match-types.md

```diff
@@ -22,16 +23,17 @@ Elem[List[Float]]  =:=  Float
 Elem[Nil.type]     =:=  Nothing
 ```
 
-Here `=:=` is understood to mean that left and right hand sides are mutually subtypes of each other.
+Here `=:=` is understood to mean that left and right hand sides are mutually
+subtypes of each other (that is `left <: right` and `right <: left`).
```

Comment:
I think I prefer it without the extra precision. That's the only meaning we have for `=:=` in Scala anyway...